pages/contact.tsx

```diff
@@ -1,3 +1,3 @@
 export default function ContactPage() {
-    return <div>Contact Page placeholder</div>
+    return <div>Contact Page holder</div>
```

Comment:
I cant believe you even wasted energy on this lmao

Reply:
I put the whole about me file in there 🤣 and commited it then im clicking the link going nuts like wtf and it was in the wrong file. so i had to re do it 🤣😂

Reply:
LOL